cpanfile

```diff
@@ -82,7 +82,7 @@ requires 'namespace::autoclean';
 requires 'Net::Amazon::S3';
 requires 'Net::DNS::Resolver';
 requires 'Net::Domain::TLD', '1.75';
-requires 'Net::Facebook::Oauth2', '0.10';
+requires 'Net::Facebook::Oauth2', '0.11';
```

Comment:
Might be useful to mention this in the release notes somewhere in case people don't always run the module updates?

Reply:
They'll have had issues many times :) https://fixmystreet.org/updating/ tells you you need to run `script/update` every time you update the code; I'd actually like that first thing top of the page, will think of some way to reword it for those who already know about git.